.github/workflows/quarkus-snapshot.yaml

```diff
@@ -15,7 +15,7 @@ env:
   # Repo specific setting #
   #########################
 
-  ECOSYSTEM_CI_REPO_PATH: quarkiverse-hibernate-types
```

Comment:
This needs to be `quarkiverse-hibernate-types` because it's the path in https://github.com/quarkusio/quarkus-ecosystem-ci/tree/main/quarkiverse-hibernate-types

Reply:
Done!